server/src/yml-objects/YmlArgument.ts

```diff
@@ -0,0 +1,22 @@
+import { CompletionItemKind } from 'vscode-languageserver';
+
+import { AbstractYmlObject, YmlType } from './AbstractYmlObject';
+import { YmlSymbolList } from './YmlSymbolList';
+
+export enum ImplicitKey {
+    EXCLUSIVE,
+    INCLUSIVE,
+}
+
+export class YmlArgument extends AbstractYmlObject {
+    public domains?: YmlType;
+    public domainsLevel2?: YmlType;
+    public symbolList?: YmlSymbolList;
+    public key?: string;
+    public implicitKey?: boolean;
+    public aloneKey?: ImplicitKey;
```

Comment:
You can _be_ alone but not _an alone something_. Maybe “lone”, but I still don't understand what this field represents so I can't really say.

Reply:
Got this from `predefinedObjects.xml`:
![image](https://user-images.githubusercontent.com/15327919/54209656-70f45500-44de-11e9-8b6c-f4b9c6c6a44b.png)

Reply:
Wut. So you have no idea what it means? :thinking:

Reply:
I guess that some keys can be used with other ones while some must be used alone?

Reply:
Dunno. This doesn't quite match with the “inclusive” / “exclusive” values. But OK…